system_tests/vision.py

```diff
@@ -427,8 +427,8 @@ def tearDown(self):
     def _assert_text(self, text):
         self.assertIsInstance(text, EntityAnnotation)
         self.assertIn(text.description, self.DESCRIPTIONS)
-        self.assertIn(text.locale, (None, 'en'))
-        self.assertNotEqual(text.score, 0.0)
+        self.assertIn(text.locale, (None, '', 'en'))
+        self.assertIsInstance(text.score, (type(None), float))
```

Comment:
I'm guessing that's what's going on. It happened to Speech a while ago too.
https://github.com/GoogleCloudPlatform/google-cloud-python/issues/2935

It seems like it's always `score`.